.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,32 +1,29 @@
-<!--- Provide a general summary of your changes in the Title above -->
-
+<!--- 
 Many thanks for submitting your Pull Request ❤️!
 
-Please make sure that your PR meets the following requirements:
-
-- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md)
-- [ ] Pull Request contains description of the issue
-- [ ] Pull Request contains link to the JIRA issue
-- [ ] Pull Request contains link to any dependent or related Pull Request
+Please complete the following sections for a smooth review.
+-->
 
 ## Description
 <!--- Describe your changes in detail -->
 
-## JIRA issue:
+## JIRA issue
```

Comment:
i may add "If applicable" for this section as well.

Reply:
Thinking of it... I am not quite sure it deserves a dedicated section. Isn't it enough to add a link in the description, preferably at the end? We actually "ensure" it is added by making it part of the checklist. WDYT?

Reply:
that would be even better.